<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ManoHarsha" failures="2" tests="4" name="parallel.ParallelRun" time="35.075" errors="0" timestamp="22 Aug 2023 16:35:29 GMT" skipped="0">
  <testcase classname="parallel.ParallelRun" name="runScenario" time="9.678">
    <failure message="expected:&lt;4&gt; but was:&lt;5&gt;" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected:<4> but was:<5>
at org.junit.Assert.fail(Assert.java:89)
at org.junit.Assert.failNotEquals(Assert.java:835)
at org.junit.Assert.assertEquals(Assert.java:647)
at org.junit.Assert.assertEquals(Assert.java:633)
at parallel.AccountsPageSteps.account_selection_count_is(AccountsPageSteps.java:69)
at ✽.account selection count is 4(file:///C:/Users/manoh/CucumberProjects/cucumber1/GIT%20Repos/cucumberJunit/CucumberTestNGParalel1/CucumberTestNGParallel/src/test/resources/parallel/AccountPage.feature:19)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="parallel.ParallelRun" name="runScenario" time="9.364"/>
  <testcase classname="parallel.ParallelRun" name="runScenario" time="7.978">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
at org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at parallel.LoginSteps.i_verify_title_should_be(LoginSteps.java:23)
at ✽.I verify title should be "Login - My Show"(file:///C:/Users/manoh/CucumberProjects/cucumber1/GIT%20Repos/cucumberJunit/CucumberTestNGParalel1/CucumberTestNGParallel/src/test/resources/parallel/loginPage.feature:9)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="parallel.ParallelRun" name="runScenario" time="8.055"/>
</testsuite> <!-- parallel.ParallelRun -->
